src/useChat.ts

```diff
@@ -34,11 +33,11 @@ type UseChatConfig = {
 
 export const useChat = ({
   customerId,
-  corpusIds,
+  corpusKey,
   apiKey,
-  enableStreaming = true,
+  numberOfSearchResults = 15,
   language = "eng",
-  enableFactualConsistencyScore,
+  enableFactualConsistencyScore = true,
```

Comment:
If this is not absolutely necessary, I'd revert this. This would potentially introduce unexpected changes in consuming code on upgrade.